pages/metamask/notification-page.js

```diff
@@ -1,5 +1,7 @@
 const notificationPage = '.notification';
 const nextButton = `${notificationPage} .permissions-connect-choose-account__bottom-buttons button:nth-child(2)`;
+const signButton = `${notificationPage} [data-testid="request-signature__sign"]`;
```

Comment:
So far I've used classes to identify elements. I'm okay with `data-testid` but I would like to be consistent. How strong do you feel about this, and are you willing to change this to use the class of the element?